tests/test_il_aging_advisory_council.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest  # noqa
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.il_aging_advisory_council import IlAgingAdvisoryCouncilSpider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "il_aging_advisory_council.html"),
+    url="https://www2.illinois.gov/aging/PartnersProviders/OlderAdult/Pages/acmeetings.aspx",
+)
+spider = IlAgingAdvisoryCouncilSpider()
+
+freezer = freeze_time("2019-12-23")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+# def test_tests():
+#     print("Please write some tests for this spider or at least disable this one.")
+#     assert False
+"""
+Uncomment below
+"""
+
+
+def test_title():
+    assert parsed_items[0]["title"] == "Illinois Department on Aging Advisory Committee Meetings"
+
+
+def test_start():
```

Comment:
Generally we'll want to keep all of the boilerplate test methods (like `test_end`) in here unless there's a reason for removing them. They're helpful for tracking changes when updating scrapers

Reply:
Updated tests so that all attributes of 'meeting' object are being tested.